src/unity/lib/image_util.cpp

```diff
@@ -230,13 +235,41 @@ std::vector<std::string> get_directory_files(std::string url, bool recursive) {
   return ret;
 }
 
+
+bool has_ending (std::string fullString, std::string ending) {
+    if (fullString.length() >= ending.length()) {
```

Comment:
I think with [rfind](http://en.cppreference.com/w/cpp/string/basic_string/rfind) the body of this function can be simplified to:

```
return fullString.rfind(ending) == fullString.length() - ending.length();
```